web_notify/static/src/js/web_client.js

```diff
@@ -57,23 +60,17 @@ WebClient.include({
     },
     show_application: function() {
         var res = this._super();
-        this.start_polling();
+        base_bus.bus.start_polling();
+        base_bus.bus.add_channel(this.channel_warning);
+        base_bus.bus.add_channel(this.channel_info);
+        base_bus.bus.on('notification', this, this.bus_notify_notification);
         return res
     },
     on_logout: function() {
-        var self = this;
-        base_bus.bus.off('notification', this, this.bus_notification);
+        base_bus.bus.off('notification', this, this.bus_notify_notification);
         this._super();
     },
-    start_polling: function() {
-        this.channel_warning = 'notify_warning_' + this.session.uid;
-        this.channel_info = 'notify_info_' + this.session.uid;
-        base_bus.bus.add_channel(this.channel_warning);
-        base_bus.bus.add_channel(this.channel_info);
-        base_bus.bus.on('notification', this, this.bus_notification);
-        base_bus.bus.start_polling();
-    },
-    bus_notification: function(notifications) {
+    bus_notify_notification: function (notifications) {
```

Comment:
I think a better name here would be `_web_notify_notification`.

Reply:
yep - good name